editor/src/components/canvas/canvas-strategies/absolute-resize-strategy.tsx

```diff
@@ -0,0 +1,148 @@
+import { isHorizontalPoint } from 'utopia-api/core'
+import { getLayoutProperty } from '../../../core/layout/getLayoutProperty'
+import { framePointForPinnedProp } from '../../../core/layout/layout-helpers-new'
+import { MetadataUtils } from '../../../core/model/element-metadata-utils'
+import { mapDropNulls } from '../../../core/shared/array-utils'
+import { isRight, right } from '../../../core/shared/either'
+import { JSXElement } from '../../../core/shared/element-template'
+import { CanvasRectangle, CanvasVector } from '../../../core/shared/math-utils'
+import { ElementPath } from '../../../core/shared/project-file-types'
+import { withUnderlyingTarget } from '../../editor/store/editor-state'
+import { stylePropPathMappingFn } from '../../inspector/common/property-path-hooks'
+import { EdgePosition } from '../canvas-types'
+import {
+  AdjustCssLengthProperty,
+  adjustCssLengthProperty,
+} from '../commands/adjust-css-length-command'
+import { updateHighlightedViews } from '../commands/update-highlighted-views-command'
+import { AbsoluteResizeControl } from '../controls/select-mode/absolute-resize-control'
+import { CanvasStrategy } from './canvas-strategy-types'
+
+type AbsolutePin = 'left' | 'top' | 'right' | 'bottom' | 'width' | 'height'
+
+export const absoluteResizeStrategy: CanvasStrategy = {
+  id: 'ABSOLUTE_RESIZE',
+  name: 'Absolute Resize',
+  isApplicable: (canvasState, _interactionState, metadata) => {
+    if (canvasState.selectedElements.length > 0) {
```

Comment:
I wonder if this logic should be shared between all the absolutely positioned strategies.